src/applications/debt-letters/components/DebtDetails.jsx

```diff
@@ -1,120 +1,154 @@
+          <a href="/debt-letters/debt-list">Your VA debt</a>
+        </Breadcrumbs>
+        <h1 className="vads-u-font-family--serif vads-u-margin-bottom--2">
+          Your {deductionCodes[selectedDebt.deductionCode]} debt
+        </h1>
+        <div className="vads-l-row">
+          <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-padding-right--2p5 vads-l-col--12 medium-screen:vads-l-col--8 vads-u-font-family--sans">
+            <p className="vads-u-font-size--h3 vads-u-font-family--serif vads-u-margin-top--0">
+              Updated on{' '}
+              {moment(last(selectedDebt.debtHistory).date).format(
+                'MMMM D, YYYY',
+              )}
+            </p>
+            <div className="vads-u-display--flex vads-u-flex-direction--row">
+              <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-margin-right--2">
+                <p className="vads-u-margin-y--0 vads-u-font-weight--bold">
+                  Date of first notice:
+                </p>
+                <p className="vads-u-margin-y--0 vads-u-font-weight--bold">
+                  Original debt amount:
+                </p>
+                <p className="vads-u-margin-y--0 vads-u-font-weight--bold">
+                  Current balance:
+                </p>
+              </div>
+              <div className="vads-u-display--flex vads-u-flex-direction--column">
+                <p className="vads-u-margin-y--0">
+                  {moment(first(selectedDebt.debtHistory).date).format(
+                    'MMMM D, YYYY',
+                  )}
+                </p>
+                <p className="vads-u-margin-y--0">
+                  {formatter.format(parseFloat(selectedDebt.originalAr))}
+                </p>
+                <p className="vads-u-margin-y--0">
+                  {formatter.format(parseFloat(selectedDebt.currentAr))}
+                </p>
+              </div>
+            </div>
 
-          <AlertBox
-            className="vads-u-margin-y--4"
-            headline="Informational backgroundOnly alert"
-            content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam id felis pulvinar ligula ultricies sollicitudin eget nec dui. Cras augue velit, pellentesque sit amet nisl ut, tristique suscipit sem. Cras sollicitudin auctor mattis."
-            status="info"
-            backgroundOnly
-          />
+            <AlertBox
+              className="vads-u-margin-y--4"
+              headline="Hidden alert"
+              content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam id felis pulvinar ligula ultricies sollicitudin eget nec dui. Cras augue velit, pellentesque sit amet nisl ut, tristique suscipit sem. Cras sollicitudin auctor mattis."
+              status="info"
+            />
+            <AdditionalInfo triggerText="Why might I have this debt?">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi
```

Comment:
Lorem ipsum?

Reply:
Indeed sir - this is all behind a feature flip and we do not have the content ready yet